backend/.env.example

```
@@ -2,38 +2,42 @@
 # EMBEDDING_DEVICE=cpu
 
-# FalkorDB Configuration
-FALKORDB_HOST=localhost
-FALKORDB_PORT=6379
-CODEGRAPH_BACKEND_URL=http://localhost:5000
-CODEGRAPH_SECRET_TOKEN=DevRAI_CodeGraph_Secret
+#1. backend only
+BACKEND_URL=http://localhost:8000
+ 
+#2. Discord-only mode (no GitHub, no Supabase)
+DISCORD_BOT_TOKEN="your_discord_bot_token"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)